specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -0,0 +1,2448 @@
+      }
+    },
+    "/analyze/jobs/{jobId}": {
+      "get": {
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "description": "Get the status of an analysis job.  A job may consist of one or more tasks.  Once all tasks are completed, the job will transition to the completed state and results will be available for each task.",
+        "operationId": "AnalyzeStatus",
+        "summary": "Get analysis status and results",
+        "parameters": [
+          {
+            "$ref": "#/parameters/AnalyzeJobId"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "default": 20,
+            "description": "(Optional) Set the maximum number of results per task. When both $top and $skip are specified, $skip is applied first.",
+            "in": "query",
+            "maximum": 50,
+            "minimum": 1,
+            "name": "$top",
+            "type": "integer"
+          },
+          {
+            "default": 0,
+            "description": "(Optional) Set the number of elements to offset in the response. When both $top and $skip are specified, $skip is applied first.",
+            "in": "query",
+            "minimum": 0,
+            "name": "$skip",
+            "type": "integer"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Analysis job status and metadata.",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeJobState"
+            }
+          },
+          "404": {
+            "description": "Job ID not found.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "500": {
+            "description": "Internal error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Analysis job status request": {
+            "$ref": ".//examples//SuccessfulAnalyzeStatusRequest.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/entities/health/jobs/{jobId}": {
+      "get": {
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "description": "Get details of the healthcare prediction job specified by the jobId.",
+        "operationId": "HealthStatus",
+        "summary": "Get healthcare analysis job status and results",
+        "parameters": [
+          {
+            "$ref": "#/parameters/JobId"
+          },
+          {
+            "default": 20,
+            "description": "(Optional) Set the maximum number of results per task. When both $top and $skip are specified, $skip is applied first.",
+            "in": "query",
+            "maximum": 50,
+            "minimum": 1,
+            "name": "$top",
```

Comment:
@suhas92 / @abhahn - Do we support `top` and `skip` in HC?

Reply:
Yes we support Top and Skip in healthcare.